tests/unit_tests/layout/test_geo_coordinates.py

```diff
@@ -0,0 +1,118 @@
+#!/usr/bin/python3
+
+import warnings
+
+import numpy as np
+import pyproj
+import pytest
+
+from simtools.layout.geo_coordinates import GeoCoordinates
+from simtools.layout.telescope_position import TelescopePosition
+
+
+def LaPalma_reference_point():
+    lapalma = TelescopePosition(name="LaPalma")
+    lapalma.set_coordinates("mercator", 28.7621661, -17.8920302, 2177.0)
+    return lapalma
+
+
+def Paranal_reference_point():
+    paranal = TelescopePosition(name="Paranal")
+    paranal.set_coordinates("mercator", -24.68342915, -70.316345, 2147.0)
+    return paranal
+
+
+def test_crs_utm():
+    geo = GeoCoordinates()
+    utm_crs = geo.crs_utm(32719)
+    assert isinstance(utm_crs, pyproj.CRS)
+    assert utm_crs.to_epsg() == 32719
+
+    utm_none = geo.crs_utm(None)
+    assert utm_none is None
+
+
+def test_crs_wgs84():
+    geo = GeoCoordinates()
+    wgs84_crs = geo.crs_wgs84()
+    assert isinstance(wgs84_crs, pyproj.CRS)
+    assert wgs84_crs.to_epsg() == 4326
+
+
+def test_crs_local():
+    geo = GeoCoordinates()
+
+    lapalma = LaPalma_reference_point()
+    lapalma_crs = geo.crs_local(lapalma)
+    assert isinstance(lapalma_crs, pyproj.CRS)
+
+    _crs_test_dict = {
+        "proj": "tmerc",
+        "lat_0": 28.7621661,
+        "lon_0": -17.8920302,
+        "k": 1.00034158569596,
+        "x_0": 0,
+        "y_0": 0,
+        "datum": "WGS84",
+        "units": "m",
+        "no_defs": None,
+        "type": "crs",
+    }
+    # ignore warnings from pyproj that te crs.to_dict()
+    # method is not sufficient to fully describe a
+    # coordinate system - this is not relevant for the
+    # simple comparison of the crs parameters
+    with warnings.catch_warnings():
+        warnings.simplefilter("ignore")
+        lapalma_dict = lapalma_crs.to_dict()
+    assert lapalma_dict["proj"] == _crs_test_dict["proj"]
+    assert lapalma_dict["k"] == pytest.approx(_crs_test_dict["k"], rel=1.0e-4)
```

Comment:
I think we missed this low accuracy requirement as well. Should probably change to `1e-6` as well.